client/state/action-types.js

```diff
@@ -20,6 +21,9 @@ export const POSTS_REQUEST_SUCCESS = 'POSTS_REQUEST_SUCCESS';
 export const PUBLICIZE_CONNECTIONS_RECEIVE = 'PUBLICIZE_CONNECTIONS_RECEIVE';
 export const PUBLICIZE_CONNECTIONS_REQUEST = 'PUBLICIZE_CONNECTIONS_REQUEST';
 export const PUBLICIZE_CONNECTIONS_REQUEST_FAILURE = 'PUBLICIZE_CONNECTIONS_REQUEST_FAILURE';
+export const RECEIVE_PUBLICIZE_CONNECTIONS = 'RECEIVE_PUBLICIZE_CONNECTIONS';
```

Comment:
I am not sure we should be adding This in the PR since I don't think we do anything with it. `RECEIVE_PUBLICIZE_CONNECTIONS` maybe something went wrong with a rebase ? The same with `RECEIVE_SITE`